be/src/exec/spill/input_stream.cpp

```diff
@@ -98,8 +98,7 @@ StatusOr<ChunkUniquePtr> UnionAllSpilledInputStream::get_next(SerdeContext& cont
 // The raw chunk input stream. all chunks are in memory.
 class RawChunkInputStream final : public SpillInputStream {
 public:
-    RawChunkInputStream(std::vector<ChunkPtr> chunks, Spiller* spiller)
-            : _chunks(std::move(chunks)), _spiller(spiller) {}
+    RawChunkInputStream(std::vector<ChunkPtr> chunks, Spiller* spiller) : _chunks(std::move(chunks)) {}
```

Comment:
seems spiller is not used, we can remove it from parameter list.

Reply:
We make the constructor consistent with the rest of the input stream, so that it can be easily dispatch with templates later on.